imagepicker/src/main/java/com/esafirm/imagepicker/features/ImagePicker.java

```diff
@@ -138,6 +138,10 @@ public ImagePicker imageLoader(ImageLoader imageLoader) {
         return this;
     }
 
+    public ImagePickerConfig config() {
```

Comment:
I think it should be `protected` unless there's another use case that need the config to be exposed to the outside